### PR TITLE
Created MyThread, MyRunnable, IncrementRunner, Main classes and added…

### PR DESCRIPTION
**Pull request by Alekearl**
… log file.

### CODE REVIEW

File: src/main/resources/log4j.properties [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Root logger option
+log4j.rootLogger=INFO, stdout, file
+# Redirect log messages to console
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.Target=System.out
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{ABSOLUTE} %-5p %c{1}:%L - %m%n
+# Redirect log messages to a log file, support file rolling.
+log4j.appender.file=org.apache.log4j.RollingFileAppender
+log4j.appender.file.File=C:\\Users\\Alexandr\\IdeaProjects\\multithreading\\src\\main\\resources\\log4j.properties
```
Comment (oroboro-cpu) [inline src/main/resources/log4j.properties]: you should add logs to another file, for example "C:\\Users\\Alexandr\\IdeaProjects\\multithreading\\log\\logInform.log

**Review by oroboro-cpu — CHANGES_REQUESTED** [review]

**Review by oroboro-cpu — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -21,6 +21,10 @@
...
     </properties>
     <dependencies>
-
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>1.2.17</version>
+        </dependency>
     </dependencies>
 </project>
```
Comment (RomanDubovskyi) [inline pom.xml]: missing empty line at the end of the file

File: src/main/resources/log4j.properties [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+log4j.appender.stdout.Target=System.out
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{ABSOLUTE} %-5p %c{1}:%L - %m%n
+# Redirect log messages to a log file, support file rolling.
+log4j.appender.file=org.apache.log4j.RollingFileAppender
+log4j.appender.file.File=C:\\Users\\Alexandr\\IdeaProjects\\multithreading\\src\\log\\log.log
+log4j.appender.file.MaxFileSize=5MB
+log4j.appender.file.MaxBackupIndex=10
+log4j.appender.file.layout=org.apache.log4j.PatternLayout
+log4j.appender.file.layout.ConversionPattern=%d{ABSOLUTE} %-5p %c{1}:%L - %m%n
```
Comment (RomanDubovskyi) [inline src/main/resources/log4j.properties]: empty line at the end

**Review by RomanDubovskyi — APPROVED** [review]

File: pom.xml (hunk 2 above)
Comment (Alekearl) [reply pom.xml]: Thanx. added.

File: src/main/resources/log4j.properties (hunk 3 above)
Comment (Alekearl) [reply src/main/resources/log4j.properties]: thanx, added.

File: src/main/resources/log4j.properties (hunk 1 above)
Comment (Alekearl) [reply src/main/resources/log4j.properties]: Thnx, changed

**Pull request merged**